lisa/operating_system.py

```diff
@@ -549,12 +550,22 @@ def _initialize_package_installation(self) -> None:
         # sub os can override it, but it's optional
         pass
 
-    def _get_package_information(self, package_name: str) -> VersionInfo:
+    def _get_package_information(self, package_name: str) -> LisaVersionInfo:
         raise NotImplementedError()
 
-    def _get_version_info_from_named_regex_match(
-        self, package_name: str, named_matches: Match[str]
-    ) -> VersionInfo:
+    def _get_version_info_from_regex(
```

Comment:
What's the new method is for? The `package_name` should be the same meaning for `raw_version`, and the `package_name` is not used.

Reply:
Just trying to extract similar code to a separate function, each os' `_get_package_information` was using similar logic, with only the regex being different